development-tools/clinic.rst

```diff
@@ -2001,6 +2001,43 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
+.. _clinic-howto-getter:
+
+How to generate a getter
+------------------------
+
+You can use the ``@getter`` directive to generate an "impl" function
+to define a getter.
```

Comment:
It might be worth it to link to the C API docs here.

https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef.get

Reply:
Done!